spec/models/maths_and_physics/eligibility_spec.rb

```diff
@@ -19,6 +24,29 @@
 
     it "returns a symbol indicating the reason for ineligibility" do
       expect(MathsAndPhysics::Eligibility.new(teaching_maths_or_physics: false).ineligibility_reason).to eq :not_teaching_maths_or_physics
+      expect(MathsAndPhysics::Eligibility.new(current_school: schools(:hampstead_school)).ineligibility_reason).to eq :ineligible_current_school
+    end
+  end
+
+  describe "#current_school_name" do
```

Comment:
This method isn't actually used at the moment, because we don't display the name of the ineligible school. (But probably we should, same as student loans, I wonder why we don't?)

Reply:
I did have it in the `_ineligibility_reason_ineligible_current_school.html.erb` but decided to take it out in favour of replicating the mockup exactly. Do we think the copy on student loans it right?

https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/app/views/student_loans/claims/_ineligibility_reason_ineligible_current_school.html.erb

Reply:
I thought `current_school_name` was used somewhere else, I remember looking:

https://github.com/DFE-Digital/dfe-teachers-payment-service/blob/master/app/views/claims/current_school.html.erb

But still happy to change the ineligble screen.

Reply:
Ah, sorry, you're totally right, the method needs to be there. And yeah, let's just replicate the wireframes for now.

Reply:
(Although @tekin suggests [here](https://github.com/DFE-Digital/dfe-teachers-payment-service/pull/623#issuecomment-555113496) we could reuse the question...)

Reply:
Yeah,  I think we've reused the question because the attribute is `current_school` so Rails looks for the `current_school.html.erb` template file.

The ineligibility reason is part of the `MathsAndPhysics::Eligibility` class though, so I think it has to live in `maths_and_physics/claim`. We might be able to share the partial in `claim` but would we be happy with that as there is seperation of eligibility/other claim views?

Reply:
Ah, yes, of course, I forgot about the ineligibility reason. I don't think we're aiming to share ineligibility partials at the moment.